wdl/gwas/regenie_sub.wdl

```diff
@@ -11,7 +11,7 @@ task step2 {
     File bgi = bgen + ".bgi"
     File sample = bgen + ".sample"
     File pred
-    File firth_list
+    File? firth_list
```

Comment:
If I'm looking at the wdl right, this file is always present, right? It gets created no matter what in step 1 (either through binary phenotype bash shenanigans or touched when using quantitative traits), and in step2 workflow it is input regardless of trait type (see [here](https://github.com/FINNGEN/regenie-pipelines/blob/e9ea2372b23ee77a7e41ba75b77fe3a1cb515339/wdl/gwas/regenie_sub.wdl#L301)  , [here](https://github.com/FINNGEN/regenie-pipelines/blob/e9ea2372b23ee77a7e41ba75b77fe3a1cb515339/wdl/gwas/regenie_sub.wdl#L309)). And since its usage is determined by the `is_binary` flag, we can drop the optional.

Reply:
Thanks, great catch! I once made a change in the upstream that requires this to be optional but then reverted. I now dropped the optional flag.